docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
also thinking it would be nice if there was a sane default for all containers set by the daemon something like 512 (more than enough, but not unlimited)